src/index.js

```diff
@@ -38,28 +38,13 @@ if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
-
-require("./models/User");
-
-app.use(require("./routes"));
 
-/// catch 404 and forward to error handler
 app.use(function(req, res, next) {
     const err = new Error("Not Found");
     err.status = 404;
     next(err);
 });
 
-/// error handlers
-
-// development error handler
-// will print stacktrace
 if (!isProduction) {
     app.use(function(err, req, res, next) {
         console.log(err.stack);
```

Comment:
Can you use a logger instead of console.log? For example: Research on using winston logger